src/pages/Homepage.js

```diff
@@ -51,8 +54,17 @@ const Homepage = (props) => {
     }
   }
   return (
-    <div>
-      <h1>Task Master</h1>
+    <Box
+      boxSize='md'
+      border='2px'
+      borderRadius='md'
+      rounded='md'
+      boxShadow='outline'
+      bg='#f5f5dc'
```

Comment:
It would be very cool to drive most of values off of the theme. This works for now though.

Reply:
Thanks